shimmy/__init__.py

```diff
@@ -3,4 +3,4 @@
 from shimmy.dm_env_wrapper import DMEnvWrapper as DMEnvWrapperV0
 from shimmy.openspiel_wrapper import OpenspielWrapper as OpenspielWrapperV0
 
-__version__ = "0.0.1a"
+__version__ = "0.0.1b"
```

Comment:
I don't think we need to increment this yet, the package hasn't been released on pypi.

Reply:
changing back